src/Search.php

```diff
@@ -75,6 +76,7 @@ class Search
      **/
     public static function show($itemtype)
     {
+        Plugin::doHook(Hooks::PRE_ITEM_LIST, ['itemtype' => $itemtype, 'options' => []]);
```

Comment:
I do not know where is the best place to put additionnal HTML for a plugin, the best way to knw that is probably to test the behaviour on different places:
 - outside `<div class="search_page row">`;
 - inside `<div class="search_page row">`;
 - inside `<div class="col search-container>`.

I guess both `Hooks::PRE_ITEM_LIST` and `Hooks::POST_ITEM_LIST` should be located in same DOM container.